influx/query_test.go

```diff
@@ -16,6 +16,66 @@ func TestConvert(t *testing.T) {
 		wantErr  bool
 	}{
 		{
+			name:     "Test field order",
+			influxQL: `SELECT "usage_idle", "usage_guest_nice", "usage_system", "usage_guest" FROM "telegraf"."autogen"."cpu" WHERE time > :dashboardTime:`,
+			want: chronograf.QueryConfig{
+				Database:        "telegraf",
+				Measurement:     "cpu",
+				RetentionPolicy: "autogen",
+				Fields: []chronograf.Field{
+					chronograf.Field{
+						Field: "usage_idle",
+						Funcs: []string{},
+					},
+					chronograf.Field{
+						Field: "usage_guest_nice",
+						Funcs: []string{},
+					},
+					chronograf.Field{
+						Field: "usage_system",
+						Funcs: []string{},
+					},
+					chronograf.Field{
+						Field: "usage_guest",
+						Funcs: []string{},
+					},
+				},
+				Tags: map[string][]string{},
+				GroupBy: chronograf.GroupBy{
+					Tags: []string{},
+				},
+			},
+		},
+		{
+			name:     "Test field function order",
+			influxQL: `SELECT mean("usage_idle"), median("usage_idle"), count("usage_guest_nice"), mean("usage_guest_nice") FROM "telegraf"."autogen"."cpu" WHERE time > :dashboardTime:`,
+			want: chronograf.QueryConfig{
+				Database:        "telegraf",
+				Measurement:     "cpu",
+				RetentionPolicy: "autogen",
+				Fields: []chronograf.Field{
+					chronograf.Field{
+						Field: "usage_idle",
+						Funcs: []string{
+							"mean",
+							"median",
```

Comment:
Hey @timraymond , yup, the old code preserved the function order as it was a slice.  However, the field names were in a map. So, this test is just to prove the order is still the same.